session_stats.go

```diff
@@ -0,0 +1,62 @@
+package delugeclient
+
+import (
+	"github.com/gdm85/go-rencode"
+)
+
+// SessionStats ...
+type SessionStats struct {
+	PayloadDownloadRate int64
+	PayloadUploadRate   int64
+	DownloadRate        int64
+	UploadRate          int64
+	TotalDownload       int64 // net.recv_bytes
+	TotalUpload         int64 // net.sent_bytes
+	// FreeSpace           int64
```

Comment:
Why are these commented?

Reply:
These fields read failed, I'll fix it soon